lib/fs.js

```diff
@@ -40,7 +40,6 @@ const isWindows = process.platform === 'win32';
 
 const DEBUG = process.env.NODE_DEBUG && /fs/.test(process.env.NODE_DEBUG);
 const errnoException = util._errnoException;
-const printDeprecation = require('internal/util').printDeprecationMessage;
```

Comment:
@ChALkeR ... https://github.com/nodejs/node/pull/6749 landed just now, which adds a ref to `internal/fs`